# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qdvdauthor/stdlib.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- qdvdauthor-0.0.10/qdvdauthor/log.cpp.orig	2005-07-31 17:52:47.000000000 +0200
+++ qdvdauthor-0.0.10/qdvdauthor/log.cpp	2005-07-31 17:41:39.000000000 +0200
@@ -1,5 +1,6 @@
 
 #include <qstring.h>
+#include <stdio.h>
 #include "log.h"
 
 Log::Log (int iLogLevel)
--- qdvdauthor-0.0.10/qdvdauthor/menupreview.cpp.orig	2005-07-31 17:51:59.000000000 +0200
+++ qdvdauthor-0.0.10/qdvdauthor/menupreview.cpp	2005-07-31 17:52:16.000000000 +0200
@@ -24,6 +24,8 @@
 #include <qtimer.h>
 #include <qimage.h>
 
+#include <stdlib.h>
+
 #include "global.h"
 #include "dvdmenuundoobject.h"
 #include "xml_dvd.h"
